packages/api/endpoints/logs.js

```diff
@@ -12,6 +12,25 @@ function convertLogLevelForQuery(query) {
   return Object.assign({}, query, { level: log.convertLogLevel(query.level) });
 }
 
+function metricsConfig() {
+  return {
+    type: '_doc',
+    index: `${process.env.stackName}-*`,
+    queryParams: { sort_by: '@timestamp' },
+    metrics: true
+  };
+}
+
+function cumulusConfig() {
```

Comment:
We talked about this in Slack but just to document here: There were issues with `process.env` vars when not in a function. I agree that it could use a better name and will update.

Reply:
I want to go back and look at why that is - changing things in the environment variables while running feels bad, but that sounds good for this PR.